lib/Twig/Extensions/Extension/I18n.php

```diff
@@ -42,3 +42,41 @@ public function getName()
         return 'i18n';
     }
 }
+
+if(!function_exists('pgettext')){
+    /**
+     * Gets a translation checking the context.
+     *
+     * @param string $context
+     * @param string $original
+     *
+     * @return string
+     */
+    function pgettext($context, $original)
```

Comment:
pgettext is default name of function in GNU Gettext, but such function was not added in php-gettext extension.
https://www.gnu.org/software/gettext/manual/html_node/Contexts.html
In C++ gettext: ```const char *pgettext (const char *msgctxt, const char *msgid);```